Lib/pdb.py

```diff
@@ -1890,11 +1997,16 @@ def set_trace(*, header=None):
 # Post-Mortem interface
 
 def post_mortem(t=None):
-    """Enter post-mortem debugging of the given *traceback* object.
+    """Enter post-mortem debugging of the given *traceback*, or *exception*
```

Comment:
should we rename the parameter `t`?

Reply:
That would be backward incompatible ... I'm always for renaming, if we do that I would even make it positional only:

```diff
def post_mortem(tb_or_exc=None, /):
```

Reply:
renamed and made posonly.

Reply:
> That would be backward incompatible ...

Good point. Leave it then.

Reply:
> renamed and made posonly.

We can't do that. We would need to add the new option, deprecate the old, and support both for now. Not worth it.

Reply:
Removed from the commit history.